bundles/af-core/src/test/resources/form/termsandconditions/test-content.json

```diff
@@ -0,0 +1,87 @@
+{
+  "termsandconditions": {
+    "showApprovalOption": "true",
+    "jcr:title": "Terms And Conditions",
+    "fd:tnc": true,
+    "showAsPopup": "true",
```

Comment:
These should be boolean instead of string, similarly showApprovalOption should be boolean